### DISCUSSION TITLE
Connecting to Snowball with S3 Adapter

### DISCUSSION THREAD

**Discussion by matadornetwork** [iterate-ch/cyberduck#16434] [Q&A]
Wondering is it possible to connect CyberDuck to a Snowball device with an S3 Adapter? We tried updating the url to be the internal IP with the 8080 port and access credentials. When we do it changes the protocol to WebDav and after connecting gives an error it cannot list an directories. Are we doing something wrong?

Any help would be appreciated!

**Comment by dkocher** [comment]
After pasting an URL in the _Hostname_ field that includes `https` as the protocol scheme it will default to WebDAV. Please switch it back to Amazon S3 and/or enter the hostname only of the snowball device.

**Reply by dkocher** [reply]
This is in #15200.
